### DISCUSSION TITLE
A Link Prefix

### DISCUSSION THREAD

**Discussion by fpolli** [tailwindlabs/tailwindcss#15198] [Ideas]
I was wondering if there is a reason TW does not have a "link" auto-prefix. The link color needs to be relative to the background and text colors so I always need a link color in my schemes. I know there is an "accent" prefix, but I also find a need for that one, too.

**Comment by denis23x** [comment]
Hi @fpolli 

I don't think there is any special intent behind this 🤷‍♂️

Just did it this way. You can create your own variations btw.
